torch_np/_detail/_reductions.py

```diff
@@ -404,3 +421,23 @@ def quantile(a_tensor, q_tensor, axis, method, keepdims=False):
     if keepdims:
         result = _util.apply_keepdims(result, ax, ndim)
     return result
+
+
+def percentile(
+    a_tensor,
+    q_tensor,
```

Comment:
Everything in this file is a tensor, so no need to repeat ourselves. You can call these `a, q` and be less verbose.